Makefile

```diff
@@ -107,3 +107,6 @@ dockerbuild: ## Runs docker build
 	docker build . --file Dockerfile --tag $(IMAGE_NAME) 
 	docker build . --file Dockerfile.gsutil --tag $(IMAGE_NAME)-gsutil
 
+checkprojects: ## Validates ./cron/projects.txt
+	cd ./scripts && go build  -o validate
```

Comment:
nit: we can just call `go run` rather than `go build` and leave this binary around?

Reply:
go run from the root folder with the subdirectory wasn't working